client/layout/masterbar/logged-in.jsx

```diff
@@ -53,13 +54,22 @@ class MasterbarLoggedIn extends React.Component {
 		hasUnseen: PropTypes.bool,
 	};
 
+	componentDidMount() {
+		// Give a chance to direct URLs to open the sidebar on page load ( eg by clicking 'me' in wp-admin ).
+		const qryString = qs.parse( document.location.search.replace( /^\?/, '' ) );
+		if ( qryString?.openSidebar === 'true' ) {
+			this.props.setNextLayoutFocus( 'sidebar' );
+		}
+	}
+
 	clickMySites = () => {
 		this.props.recordTracksEvent( 'calypso_masterbar_my_sites_clicked' );
 		if ( ! config.isEnabled( 'nav-unification' ) ) {
 			this.props.setNextLayoutFocus( 'sidebar' );
 		} else if ( 'sites' !== this.props.section || 'sidebar' === this.props.currentLayoutFocus ) {
 			// when my-sites is not focused or sidebar is open, focus to my-sites' content. Else, open my-sites' sidebar.
 			this.props.setNextLayoutFocus( 'content' );
+			this.props.setNextLayoutFocus( 'sidebar' );
```

Comment:
We now have three different conditionals that all end up setting the layout focus to the sidebar. Can we simplify this?

Reply:
Sadly, I haven't found a way to simplify them. Bare in mind that 'nav-unification' condition will be gone in some months. I have though, dried up the code here f82036e.